utils/mongo-utility.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+var mongoose = require('mongoose');
+
+var demoSchema;
+var DemoModel;
+
+var db = mongoose.connection;
+db.on('error', console.error.bind(console, 'connection error:'));
+db.once('open', function() {
+  console.log('connection established');
+
+  // create schema
+  demoSchema = mongoose.Schema({
+    Category: {
+      type: String,
+      required: true
+    },
+    CategoryID: {
+      type: String,
+      required: true
+    },
+    MetaTitle: String,
+    MetaDescription: String
+  });
```

Comment:
Could  you please move this schema object to a file. It would be more better if you can place those schema files in a directory.